flask_nemo/__init__.py

```diff
@@ -494,10 +523,12 @@ def r_passage(self, collection, textgroup, work, version, passage_identifier):
 
         passage = self.transform(edition, text.xml)
         prev, next = self.getprevnext(text, Nemo.prevnext_callback_generator(text))
+        urn = self.transform_urn(text.urn)
         return {
             "template": self.templates["text"],
             "version": edition,
             "text_passage": Markup(passage),
+            "urn" : Markup(urn),
```

Comment:
Yes, because I want the function to be able to return html markup (such as a link .. see https://github.com/PerseusDL/Nemo/commit/bcde3cd071bdc1e1fb85f678ae9f44e2550b45a9)